### ISSUE TITLE
Print mismatch between suggested and actual filename

### ISSUE THREAD

**Issue by rlsolomon** [brett8883/DJI_Super-Patcher#122]
My run told me to load "MavicPro_SP_2.0_standard__dji_system.bin" with DUMLdore but the actual created file was "MavicPro_SP_2.0_standard_03.02.44.22_dji_system.bin".

I believe this is due to variantchooser.cmd setting the fw2 variable using the fc variable but defining fc *after* use:

Line 94: set fw2=%AC%_SP_2.0_%variant%_%fc%_dji_system.bin
Line 95: set fc=%fcbase%%varchoice%

I think reversing the order of the lines would solve the issue, but I didn't re-run to confirm.

**Comment by brett8883** [comment]
Yep I’m sure your right another one I’ll fix in the next one thanks.
